NickvisionTagger.Shared/Controllers/MainWindowController.cs

```diff
@@ -960,8 +961,8 @@ public async Task SubmitToAcoustIdAsync(string? recordingID)
                     }
                     else
                     {
-                        ratio = Fuzz.PartialRatio(value, propertyMap.Filename);
-                        if(ratio < 60)
+                        ratio = Fuzz.PartialRatio(value.Normalize(NormalizationForm.FormKD), propertyMap.Filename.Normalize(NormalizationForm.FormKD));
```

Comment:
It would be lovely if all this could be deduplicated

Reply:
Ok I factored out the check into a method and cleaned it up.